com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -1067,5 +1118,37 @@ private void DebugDisplayStore()
             table += "======\n";
             Debug.Log(table);
         }
+
+        static internal SnapshotDespawnCommand GetDespawnCommand()
```

Comment:
I don't see what this does. The struct constructor already initializes everything as default.

Reply:
Like this is a struct it will not pass the static instance by reference here.

Reply:
Right. After re-trying seems like I was confused with C++ (again).
Was using `SnapshotDespawnCommand command;` instead of
`SnapshotDespawnCommand command = new SnapshotDespawnCommand();`
will fix.

Reply:
You can also do `SnapshotDespawnCommand command = default` (will do exactly the same as default constructor)